UPGRADE_5.0.md

```diff
@@ -280,3 +280,17 @@ Backend\Core\Engine\Base\AjaxAction and Frontend\Core\Engine\Base\AjaxAction no
 | self::ERROR           | Response::HTTP_INTERNAL_SERVER_ERROR |
 
 The response class has many more constants like this for all the other status codes
+
+## BackendNavigation
+
+`Backend\Core\Engine\Navigation` also known as the service `navigation` when you are in the backend was stripped of some functions that where available because it extended a base class. 
+This is no longer the case.
+The dropped methods are:
+* getAction
```

Comment:
It is in the table below, but I've added a clarification for that below
c84fa7f50